test/functional/page_objects/dashboard_page.ts

```diff
@@ -425,7 +425,6 @@ export function DashboardPageProvider({ getService, getPageObjects }: FtrProvide
         { name: PIE_CHART_VIS_NAME, description: 'PieChart' },
         { name: 'Visualization☺ VerticalBarChart', description: 'VerticalBarChart' },
         { name: AREA_CHART_VIS_NAME, description: 'AreaChart' },
-        { name: 'Visualization☺漢字 DataTable', description: 'DataTable' },
```

Comment:
I'm not sure if this was to purposely test uncommon characters... but that should be more explicitly tested if so, so I'm not worried about removing.

Reply:
yeah me too. I think there should be some specific uncommon characters or language testing. 
But I think when I reproduce the functional tests of dashboard_page, I will give it a same title.